.github/workflows/e2e.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Run your tests
         working-directory: ./test/
         # https://github.com/actions/runner/issues/241#issuecomment-577360161
-        run: script -e -c "docker-compose exec testing pytest --base-url http://jb-com --junitxml report.xml"
+        run: script -e -c "docker-compose exec testing pytest -m 'not periodic' --base-url http://jb-com --junitxml report.xml"
```

Comment:
I didn't even think about that because I was using the local testing to work on the this test 😆 I'll throw that in

Reply:
Ya, I honestly don't know either maybe @gerbrent could give some feedback from the JB side if/when they'd like to be notified (in the future)?

Mainly just tossing it out as a question, because I don't know either :grin: